pkg/ioutils/buffer_test.go

```diff
@@ -17,7 +17,7 @@ func TestFixedBufferCap(t *testing.T) {
 func TestFixedBufferLen(t *testing.T) {
 	buf := &fixedBuffer{buf: make([]byte, 0, 10)}
 
-	buf.Write([]byte("hello"))
+	_, _ = buf.Write([]byte("hello"))
```

Comment:
Are linters really picky about this?
IMO, it only clutters the code if the assignment to the blank identifier doesn't prevent a compiler error.

Reply:
I'm sure goland is picky about it

Reply:
We disabled most linters around this. It's somewhat noisy to add the `_`, but OTOH, it makes it explicit (and prefer it over a `//nolint` comment)